webapp/tests/karma/ts/modules/contacts/contacts-content.component.spec.ts

```diff
@@ -240,4 +294,163 @@ describe('Contacts content component', () => {
     });
   });
 
+  describe('Action bar', () => {
+    it('should initialise action bar', fakeAsync(() => {
+      flush();
+
+      expect(globalActions.setRightActionBar.callCount).to.equal(1);
+      expect(globalActions.setRightActionBar.args[0][0].canDelete).to.equal(false);
+      expect(globalActions.setRightActionBar.args[0][0].canEdit).to.equal(false);
+      expect(globalActions.setRightActionBar.args[0][0].openContactMutedModal).to.be.a('function');
+      expect(globalActions.setRightActionBar.args[0][0].openSendMessageModal).to.be.a('function');
+      expect(globalActions.setRightActionBar.args[0][0].relevantForms.length).to.equal(0);
+      expect(globalActions.setRightActionBar.args[0][0].sendTo).to.deep.equal({ phone: '123', muted: true });
+    }));
+
+    it('should enable edit and delete in the right action bar', fakeAsync(() => {
+      sinon.resetHistory();
+      sessionService.isAdmin.returns(true);
+      store.overrideSelector(Selectors.getSelectedContactChildren, {
+        type: { person: true },
+        doc: { phone: '11', muted: true },
+        summary: { context: 'test' },
+        children: [
+          { _id: '1', contacts: [], type: {} },
+          { _id: '2', type: {} }
+        ]
+      });
+      store.overrideSelector(Selectors.getSelectedContactDoc, { phone: '11', muted: true });
+      store.overrideSelector(Selectors.getSelectedContactChildren, [
+        { _id: '1', contacts: [], type: {} },
+        { _id: '2', type: {} }
+      ]);
+      store.refreshState();
+      fixture.detectChanges();
+
+      flush();
+
+      expect(globalActions.setRightActionBar.callCount).to.equal(1);
+      expect(globalActions.setRightActionBar.args[0][0].canDelete).to.equal(true);
+      expect(globalActions.setRightActionBar.args[0][0].canEdit).to.equal(true);
+    }));
+
+    it('should disable edit when user isnt admin and facility is its home place ', fakeAsync(() => {
```

Comment:
I think you should another test that is the opposite of this and shows the edit button is enabled when expected.

Reply:
oh yes! Added thank you